<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractIpWidget</class>
 <widget class="QWidget" name="AbstractIpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="addressInputLabel">
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="addressInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="cidrLabel">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="cidrInput">
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>478</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="subnetCountInputLabel">
     <property name="text">
      <string>Subnet Count</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="subnetCountInput">
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="calculateBtn">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTreeWidget" name="subnetsTable">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>140</number>
     </attribute>
     <column>
      <property name="text">
       <string>No.</string>
      </property>
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
